packages/react-percy-webpack/src/compile/babelConfig.js

```diff
@@ -0,0 +1,40 @@
+export default {
```

Comment:
We now automatically provide a robust babel configuration. Users with a `.babelrc` will get their plugins and presets included as well, but for users without one everything will still work properly.